packages/portal-proto/src/features/proteinpaint/SequenceReadWrapper.tsx

```diff
@@ -34,7 +34,7 @@ export const SequenceReadWrapper: FC<PpProps> = (props: PpProps) => {
   useEffect(
```

Comment:
we have been using `useDeepCompareEffect`, especially when the dependencies are objects. I suggest using it here. The library's homepage is: https://github.com/sandiiarov/use-deep-compare#readme

Reply:
Ok, thanks, I've replaced with `useDeepCompareEffect()`.

Reply:
Thank you. I will merge it now